README.md

```diff
@@ -118,7 +118,7 @@ $ yarn
 
  Add SSL certificate:
 
- The HTTPS server looks for privatekey.pem and certificate.pem. You can use a real certificate or generate a self-signed one by following the steps below.
+ If you don't plan on using Mozilla's provided tunneling service to set up a `*.mozilla-iot.org` domain, you can use your own SSL certificate. The HTTPS server looks for `privatekey.pem` and `certificate.pem`. You can use a real certificate or generate a self-signed one by following the steps below.
```

Comment:
Doesn't it look for `ssl/privatekey.pem`?

Reply:
At the moment, yes. After my other PR gets merged, no... I'll update the README in the other PR with the new location.